src/EnergyPlus/HeatingCoils.cc

```diff
@@ -1391,14 +1442,10 @@ namespace HeatingCoils {
         using EMSManager::CheckIfNodeSetPointManagedByEMS;
 
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-        int AirInletNode;  // coil air inlet node number
-        int AirOutletNode; // coil air outlet node number
-        int ControlNode;   // coil control node number
-        int RackNum;       // Index to refrigerated case compressor rack
-        int CondNum;       // Index to refrigeration condenser
-        int DXCoilNum;     // Index to DX cooling coil
+        int CondNum;   // Index to refrigeration condenser
+        int DXCoilNum; // Index to DX cooling coil
 
-        auto &HeatingCoil(state.dataHeatingCoils->HeatingCoil);
+        auto &heatingCoil = state.dataHeatingCoils->HeatingCoil(CoilNum);
```

Comment:
Is &heatingCoil(something(CoilNum)) the same as &heatingCoil = something(CoilNum) or does one of these make a copy?